packages/engine/src/mocap/solveMotionCapturePose.ts

```diff
@@ -394,19 +394,19 @@ export function solveMotionCapturePose(
     //check state, if we are still not set to track lower body, update that
     if (!MotionCaptureRigComponent.solvingLowerBody[entity]) {
       MotionCaptureRigComponent.solvingLowerBody[entity] = 1
-      //zero bone quats to filter them out in motion capture system
-      for (const boneName of lowerBody) {
-        //only leg bones
-        MotionCaptureRigComponent.rig[boneName].x[entity] = 0
-        MotionCaptureRigComponent.rig[boneName].y[entity] = 0
-        MotionCaptureRigComponent.rig[boneName].z[entity] = 0
-        MotionCaptureRigComponent.rig[boneName].w[entity] = 0
-      }
     }
   } else {
     if (MotionCaptureRigComponent.solvingLowerBody[entity]) {
       MotionCaptureRigComponent.solvingLowerBody[entity] = 0
     }
+    //zero bone quats to filter them out in motion capture system
+    for (const boneName of lowerBody) {
+      //only leg bones
+      MotionCaptureRigComponent.rig[boneName].x[entity] = 0
+      MotionCaptureRigComponent.rig[boneName].y[entity] = 0
+      MotionCaptureRigComponent.rig[boneName].z[entity] = 0
+      MotionCaptureRigComponent.rig[boneName].w[entity] = 0
```

Comment:
should we be setting `w` to 1 instead of 0?

Reply:
At the moment, a zeroed out quaternion acts as a flag for the motion capture system that prevents a quaternion from being written to the rig. No quaternions written means the animation mixer will take over and these bones will just be animated.